.travis.yml

```diff
@@ -1,324 +1 @@
-language: python
-dist: xenial
-
-cache:
-    directories:
-        - $HOME/.cache/pip
-
-before_script:
-  # On Travis, the fastest parallelization for integration tests has proved to be 4.
-  - 'if [[ "$TOXENV" == *"integration"* ]]; then export PYTEST_ADDOPTS="--numprocesses 4"; fi'
-  # Use Travis retry feature for farm tests since they are flaky
-  - 'if [[ "$TOXENV" == "travis-test-farm"* ]]; then export TRAVIS_RETRY=travis_retry; fi'
-  - export TOX_TESTENV_PASSENV=TRAVIS
-  - 'if [[ "$SNAP" == true ]]; then snap/local/build_and_install.sh $TARGET_ARCH; fi'
-
-# Only build pushes to the master branch, PRs, and branches beginning with
-# `test-`, `travis-test-`, or of the form `digit(s).digit(s).x` or
-# `vdigit(s).digit(s).digit(s)`. As documented at
-# https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches,
-# this includes tags so pushing tags of the form `vdigit(s).digit(s).digit(s)`
-# will also trigger tests. This reduces the number of simultaneous Travis runs,
-# which speeds turnaround time on review since there is a cap of on the number
-# of simultaneous runs.
-branches:
-  # When changing these branches, please ensure the documentation under
-  # "Running tests in CI" is still correct.
```

Comment:
I removed the references to Travis in `contributing.rst`. I also disabled full tests on Azure for branches starting with `azure-test`, since I think `test-` is sufficient now with a unique CI system.

Reply:
Running `git grep -i travis`, I think the other files with Travis references we should probably really fix now rather than later are:

- certbot/README.rst
- tests/lock_test.py
- tests/modification-check.py

If you want to leave the others, you can, but can you create an issue to track removing them?

Reply:
For the issue: https://github.com/certbot/certbot/issues/8120

Reply:
Done.

For the readme, I targeted the `full-test-suite`, which I believe is the most significant to assess the global stability of Certbot.